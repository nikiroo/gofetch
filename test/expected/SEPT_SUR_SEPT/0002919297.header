0Procès du sous-marin danois: Peter Madsen condamné à perpétuité	null/SEPT_SUR_SEPT/0002919297		70
i  [Monde] 26-09-2018 14:37
i
i    La  Haute  cour  de  Copenhague a confirmé mercredi en appel la
i    condamnation  de l'inventeur danois Peter Madsen à la réclusion
i    à perpétuité pour ...
i
