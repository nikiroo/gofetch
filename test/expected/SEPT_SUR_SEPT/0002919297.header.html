<!DOCTYPE html>
<html>
<head>
  <meta http-equiv='content-type' content='text/html; charset=utf-8'>
  <meta name='viewport' content='width=device-width, initial-scale=1.0'>
  <style type='text/css'>
    body { margin: 1em 15%; }
  </style>
</head>
<body>
<div class='story-header'>
	<h2><a href='0002919297.html'>Procès du sous-marin danois: Peter Madsen condamné à perpétuité</a></h2>
	<div class='details'>([Monde] 26-09-2018 14:37)</div>
	<br/>
	<div class='content' style='text-align: justify'>
		La Haute cour de Copenhague a confirmé mercredi en appel la condamnation de l%27inventeur danois Peter Madsen à la réclusion à perpétuité pour ...
	</div>
<hr/>
</div>
</body>
